.github/workflows/ci-precommit.yml

```diff
@@ -17,8 +17,8 @@
 
 name: CI - Precommit
 on:
-  pull_request:
-    branches: [main]
+  push:
```

Comment:
I suggest you revert these commits. You can open a pull request against `main` in your fork to trigger CI also.

Reply:
Otherwise we cannot run CI upstream.

Reply:
Reverted :)